app/config/config.yml

```diff
@@ -180,16 +180,16 @@ swiftmailer:
   username: "%mailer.user%"
   password: "%mailer.password%"
 
-# Translation configuration
-jms_translation:
-  configs:
-    app:
-      dirs: ["%kernel.root_dir%", "%kernel.root_dir%/../src"]
-      output_dir: "%kernel.root_dir%/Resources/translations"
-      ignored_domains: [routes]
-      excluded_names: ["*TestCase.php", "*Test.php", "DefaultMenuListener.php"]
-      excluded_dirs: [cache, data, logs]
-      extractors: [jms_i18n_routing]
+## Translation configuration
```

Comment:
just remove this? as far as I know this isn't used anywhere?

Reply:
I'm confused with your question marks. AFAIK none of us uses the bundle anymore, there's not even support for Symfony 4 yet as far as I could tell. I'll just remove the comments.

Reply:
Nvm, I saw the first comment line and though you just put it in comments, but you actually deleted it, so all ok!